src/Playwright.Tests/BrowserTypeConnectTests.cs

```diff
@@ -80,71 +82,396 @@ public void TearDown()
+                var context = await browser.NewContextAsync();
+                var page = await context.NewPageAsync();
+                var result = await page.EvaluateAsync("11 * 11");
+                Assert.AreEqual(result.ToString(), "121");
+                await browser.CloseAsync();
+            }
+            {
+                var browser = await BrowserType.ConnectAsync(_browserServer.WSEndpoint);
+                var context = await browser.NewContextAsync();
+                var page = await context.NewPageAsync();
+                await page.GotoAsync(Server.EmptyPage);
+                Assert.AreEqual(Server.EmptyPage, page.Url);
+                await browser.CloseAsync();
+            }
         }
 
         [PlaywrightTest("browsertype-connect.spec.ts", "should be able to connect two browsers at the same time")]
-        [Ignore("SKIP WIRE")]
-        public void ShouldBeAbleToConnectTwoBrowsersAtTheSameTime()
+        public async Task ShouldBeAbleToConnectTwoBrowsersAtTheSameTime()
+        {
+            var browser1 = await BrowserType.ConnectAsync(_browserServer.WSEndpoint);
+            Assert.AreEqual(browser1.Contexts.Count, 0);
+            await browser1.NewContextAsync();
+            Assert.AreEqual(browser1.Contexts.Count, 1);
+
+            var browser2 = await BrowserType.ConnectAsync(_browserServer.WSEndpoint);
+            Assert.AreEqual(browser2.Contexts.Count, 0);
+            var context2 = await browser2.NewContextAsync();
+            Assert.AreEqual(browser1.Contexts.Count, 1);
+            Assert.AreEqual(browser2.Contexts.Count, 1);
+
+            await browser1.CloseAsync();
+            Assert.AreEqual(browser2.Contexts.Count, 1);
+
+            var page2 = await context2.NewPageAsync();
+            var result = await page2.EvaluateAsync("11 * 11");
+            Assert.AreEqual(result.ToString(), "121");
+            await browser2.CloseAsync();
+        }
+
+        [PlaywrightTest("browsertype-connect.spec.ts", "should timeout in connect while connecting")]
+        [Skip(SkipAttribute.Targets.Windows)]
+        public async Task ShouldTimeoutInConnectWhileConnecting()
+        {
+            var exception = await PlaywrightAssert.ThrowsAsync<PlaywrightException>(async () => await BrowserType.ConnectAsync(_browserServer.WSEndpoint, new BrowserTypeConnectOptions { Timeout = 1 }));
+            StringAssert.Contains("Timeout 1ms exceeded", exception.Message);
+        }
+
+        [PlaywrightTest("browsertype-connect.spec.ts", "should support slowmo option")]
+        public async Task ShouldSupportSlowMo()
         {
+            var browser = await BrowserType.ConnectAsync(_browserServer.WSEndpoint, new BrowserTypeConnectOptions { SlowMo = 200 });
+            var context = await browser.NewContextAsync();
+            var time1 = DateTime.Now;
+            var page = await context.NewPageAsync();
+            await page.GotoAsync(Server.EmptyPage);
```

Comment:
I don't see these lines in the original test.

Reply:
Have updated it.